Gopkg.lock

```diff
@@ -44,7 +56,8 @@
 [[projects]]
   name = "github.com/marstr/guid"
   packages = ["."]
-  revision = "8bdf7d1a087ccc975cf37dd6507da50698fd19ca"
+  revision = "8bd9a64bf37eb297b492a4101fb28e80ac0b290f"
```

Comment:
I think there is an item #949. is anyone doing this ? Otherwise can you pick it up ? If you  don't have time just assign it to me.

Reply:
Interesting. So when we removed the references to Satori's package, did we replace them with Martin's package?

Reply:
@mcardosos there was only 1 place where we were actually generating a UUID so to make that the behavior is consistent for people who use go get or not, we switched to that temporarily yes

Reply:
OKok, pushed changes about the guid issue here.
Release 1.2.0 from satori's package removed breaking changes.
This is being tracked here https://github.com/satori/go.uuid/issues/66